marketplace/collect.schema.json

```diff
@@ -1,8 +1,6 @@
 {
-    "$id": "https://schemas.aramis.group/marketplace/collect.schema.json",
```

Comment:
Maybe you could leave the `$id` attribute. It's a way to declare the perma-link of the file